index.html

```diff
@@ -226,7 +226,9 @@ <h3>
 href="https://identity.foundation/bbs-signature/">IETF BBS+ RFC</a> is published
 in time. Concrete serializations for VC-JWP might be provided, if an <a
 href="https://json-web-proofs.github.io/json-web-proofs/">IETF JWP RFC</a> is
-published in time. The final output specifications might not map directly to
+published in time. 
+Concrete serializations for <a href="https://github.com/w3c-ccg/lds-jws2020/">Linked Data Signatures for JWS</a>.
```

Comment:
Add "will be provided" to the end of this sentence fragment so that it becomes a sentence.  Then I'd be glad to approve.

Reply:
Ditto :+1:

Reply:
should be in with 334d442 now